integration/spring-data/testsuite/webmvc/pom.xml

```diff
@@ -125,6 +125,56 @@
         </dependency>
     </dependencies>
 
+    <build>
```

Comment:
The `<id>hibernate-apt</id>` profile already includes the annotation processor for testing. Is that not enough?

Reply:
This is about the entity-view-processor, not about the Hibernate static metamodel.

Reply:
That profile also generate the entity view static metamodel classes.